example_3 launch errors

example_3 has strange errors in the CI on ros2_control_ci stack builds
https://github.com/ros-controls/ros2_control_ci/actions/runs/13404066860/job/37440624767

```
    [INFO] [launch]: All log files can be found below /root/.ros/log/2025-02-19-03-06-48-190235-157a4a21df86-46690
    [INFO] [launch]: Default logging verbosity is set to INFO
    [INFO] [ros2_control_node-1]: process started with pid [46707]
    [INFO] [robot_state_publisher-2]: process started with pid [46708]
    [INFO] [spawner-3]: process started with pid [46709]
    [robot_state_publisher-2] [INFO] [1739934408.793304335] [robot_state_publisher]: Robot initialized
    [ros2_control_node-1] [INFO] [1739934408.804324926] [controller_manager]: Subscribing to '/robot_description' topic for robot description.
    [ros2_control_node-1] [INFO] [1739934408.816461424] [controller_manager]: update rate is 100 Hz
    [ros2_control_node-1] [INFO] [1739934408.816487041] [controller_manager]: Spawning controller_manager RT thread with scheduler priority: 50
    [ros2_control_node-1] [WARN] [1739934408.816570645] [controller_manager]: Could not enable FIFO RT scheduling policy: with error number <1>(Operation not permitted). See [https://control.ros.org/master/doc/ros2_control/controller_manager/doc/userdoc.html] for details on how to enable realtime scheduling.
    [ros2_control_node-1] [INFO] [1739934408.931051891] [controller_manager]: Received robot description from topic.
    [ros2_control_node-1] [INFO] [1739934408.937498854] [controller_manager.resource_manager]: Loading hardware 'RRBotSystemMultiInterface' 
    [ros2_control_node-1] [INFO] [1739934408.938836083] [controller_manager.resource_manager]: Loaded hardware 'RRBotSystemMultiInterface' from plugin 'ros2_control_demo_example_3/RRBotSystemMultiInterfaceHardware'
    [ros2_control_node-1] [INFO] [1739934408.939110617] [controller_manager.resource_manager]: Initialize hardware 'RRBotSystemMultiInterface' 
    [ros2_control_node-1] [INFO] [1739934408.939555542] [controller_manager.resource_manager]: Successful initialization of hardware 'RRBotSystemMultiInterface'
    [ros2_control_node-1] [INFO] [1739934408.940651337] [resource_manager]: 'configure' hardware 'RRBotSystemMultiInterface' 
    [ros2_control_node-1] [INFO] [1739934408.940683027] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Configuring ...please wait...
    [ros2_control_node-1] [INFO] [1739934408.940743801] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Successfully configured!
    [ros2_control_node-1] [INFO] [1739934408.940758138] [resource_manager]: Successful 'configure' of hardware 'RRBotSystemMultiInterface'
    [ros2_control_node-1] [INFO] [1739934408.940781321] [resource_manager]: 'activate' hardware 'RRBotSystemMultiInterface' 
    [ros2_control_node-1] [INFO] [1739934408.940788665] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Activating... please wait...
    [ros2_control_node-1] [INFO] [1739934408.940801830] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: System successfully activated! 0
    [ros2_control_node-1] [INFO] [1739934408.940815876] [resource_manager]: Successful 'activate' of hardware 'RRBotSystemMultiInterface'
    [ros2_control_node-1] [INFO] [1739934408.940854518] [controller_manager]: Resource Manager has been successfully initialized. Starting Controller Manager services...
    [ros2_control_node-1] [INFO] [1739934408.946778557] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934408.946946182] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Writing commands:
    [ros2_control_node-1] 	command pos: 0.00, vel: 0.00, acc: 0.00 for joint 0, control lvl: 0
    [ros2_control_node-1] 	command pos: 0.00, vel: 0.00, acc: 0.00 for joint 1, control lvl: 0
    [ros2_control_node-1] [INFO] [1739934408.956717279] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934408.966911821] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934408.976955991] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934408.986703996] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934408.996718087] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.006720270] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.016719817] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.026719202] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.036762651] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.047085667] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [spawner-3] [INFO] [1739934409.049102327] [spawner_forward_velocity_controller]: [NON-XML-CHAR-0x1B][96mSetting controller param "params_file" to "['/home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml']" for [NON-XML-CHAR-0x1B][1mforward_velocity_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1739934409.058171721] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [spawner-3] [INFO] [1739934409.062640593] [spawner_forward_velocity_controller]: [NON-XML-CHAR-0x1B][96mSetting controller param "type" to "velocity_controllers/JointGroupVelocityController" for [NON-XML-CHAR-0x1B][1mforward_velocity_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1739934409.067216046] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.075582073] [controller_manager]: Loading controller : 'forward_velocity_controller' of type 'velocity_controllers/JointGroupVelocityController'
    [ros2_control_node-1] [INFO] [1739934409.075625394] [controller_manager]: Loading controller 'forward_velocity_controller'
    [ros2_control_node-1] [INFO] [1739934409.076730642] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.082381769] [controller_manager]: Controller 'forward_velocity_controller' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.086853700] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.096762572] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.107158613] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.116896228] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.126722038] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.127716073] [controller_manager]: Loading controller : 'joint_state_broadcaster' of type 'joint_state_broadcaster/JointStateBroadcaster'
    [ros2_control_node-1] [INFO] [1739934409.127748804] [controller_manager]: Loading controller 'joint_state_broadcaster'
    [spawner-3] [INFO] [1739934409.129039435] [spawner_forward_velocity_controller]: [NON-XML-CHAR-0x1B][94mLoaded [NON-XML-CHAR-0x1B][1mforward_velocity_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1739934409.136974660] [controller_manager]: Controller 'joint_state_broadcaster' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.137327862] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.146725158] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.156720136] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.157188538] [controller_manager]: Loading controller : 'rrbot_with_sensor_position_controller' of type 'forward_command_controller/ForwardCommandController'
    [ros2_control_node-1] [INFO] [1739934409.157214046] [controller_manager]: Loading controller 'rrbot_with_sensor_position_controller'
    [ros2_control_node-1] [INFO] [1739934409.157420894] [controller_manager]: Controller 'rrbot_with_sensor_position_controller' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_13/share/ros2_control_demo_example_13/config/three_robots_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.166880372] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.176717373] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.176994238] [controller_manager]: Loading controller : 'threedofbot_joint_state_broadcaster' of type 'joint_state_broadcaster/JointStateBroadcaster'
    [ros2_control_node-1] [INFO] [1739934409.177018413] [controller_manager]: Loading controller 'threedofbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.179544035] [controller_manager]: Controller 'threedofbot_joint_state_broadcaster' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_13/share/ros2_control_demo_example_13/config/three_robots_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.186967289] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.196717598] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.197107554] [controller_manager]: Loading controller : 'rrbot_with_sensor_joint_state_broadcaster' of type 'joint_state_broadcaster/JointStateBroadcaster'
    [ros2_control_node-1] [INFO] [1739934409.197135817] [controller_manager]: Loading controller 'rrbot_with_sensor_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.198740537] [controller_manager]: Controller 'rrbot_with_sensor_joint_state_broadcaster' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_13/share/ros2_control_demo_example_13/config/three_robots_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.206727423] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.216721800] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.217331265] [controller_manager]: Configuring controller: 'forward_velocity_controller'
    [ros2_control_node-1] [INFO] [1739934409.219745502] [forward_velocity_controller]: configure successful
    [ros2_control_node-1] [INFO] [1739934409.226716858] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.227146585] [controller_manager]: Configuring controller: 'joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.227240492] [joint_state_broadcaster]: 'joints' or 'interfaces' parameter is empty. All available state interfaces will be published
    [ros2_control_node-1] [INFO] [1739934409.236728467] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.246718156] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Nothing is using the hardware interface!
    [ros2_control_node-1] [INFO] [1739934409.247411144] [controller_manager]: Activating controllers: [ forward_velocity_controller ]
    [ros2_control_node-1] [INFO] [1739934409.256768438] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Reading states:
    [ros2_control_node-1] 	pos: 0.00, vel: 0.00, acc: 0.00 for joint 0
    [ros2_control_node-1] 	pos: 0.00, vel: 0.00, acc: 0.00 for joint 1
    [ros2_control_node-1] [INFO] [1739934409.257169550] [forward_velocity_controller]: activate successful
    [ros2_control_node-1] [INFO] [1739934409.267220105] [controller_manager]: Activating controllers: [ joint_state_broadcaster ]
    [spawner-3] [INFO] [1739934409.268397119] [spawner_forward_velocity_controller]: [NON-XML-CHAR-0x1B][92mConfigured and activated [NON-XML-CHAR-0x1B][1mforward_velocity_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1739934409.287235745] [controller_manager]: Configuring controller: 'rrbot_with_sensor_position_controller'
    [ros2_control_node-1] [INFO] [1739934409.288469059] [rrbot_with_sensor_position_controller]: configure successful
    [ros2_control_node-1] [INFO] [1739934409.297345027] [controller_manager]: Configuring controller: 'threedofbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.297468869] [threedofbot_joint_state_broadcaster]: Publishing state interfaces defined in 'joints' and 'interfaces' parameters.
    [ros2_control_node-1] [WARN] [1739934409.297511219] [threedofbot_joint_state_broadcaster]: Mapping from 'position' to interface 'position' will not be done, because 'position' is defined in 'interface' parameter.
    [ros2_control_node-1] [INFO] [1739934409.308178210] [controller_manager]: Configuring controller: 'rrbot_with_sensor_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.308285652] [rrbot_with_sensor_joint_state_broadcaster]: Publishing state interfaces defined in 'joints' and 'interfaces' parameters.
    [ros2_control_node-1] [WARN] [1739934409.308328923] [rrbot_with_sensor_joint_state_broadcaster]: Mapping from 'position' to interface 'position' will not be done, because 'position' is defined in 'interface' parameter.
    [ros2_control_node-1] [INFO] [1739934409.397505773] [controller_manager]: Loading controller : 'threedofbot_position_controller' of type 'forward_command_controller/ForwardCommandController'
    [ros2_control_node-1] [INFO] [1739934409.397544555] [controller_manager]: Loading controller 'threedofbot_position_controller'
    [ros2_control_node-1] [INFO] [1739934409.397799824] [controller_manager]: Controller 'threedofbot_position_controller' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_13/share/ros2_control_demo_example_13/config/three_robots_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.417109550] [controller_manager]: Activating controllers: [ rrbot_with_sensor_joint_state_broadcaster ]
Error: s2_control_node-1] [ERROR] [1739934409.426840997] [controller_manager]: Caught exception of type : St13runtime_error while claiming the state interfaces. Can't activate controller 'rrbot_with_sensor_joint_state_broadcaster': State interface with key 'rrbot_with_sensor_joint1/position' does not exist
Error: s2_control_node-1] [ERROR] [1739934409.427020621] [controller_manager]: Could not activate controller : 'rrbot_with_sensor_joint_state_broadcaster'
    [INFO] [spawner-3]: process has finished cleanly [pid 46709]
    [INFO] [spawner-4]: process started with pid [46781]
    [ros2_control_node-1] [INFO] [1739934409.436971455] [controller_manager]: Loading controller : 'rrbot_joint_state_broadcaster' of type 'joint_state_broadcaster/JointStateBroadcaster'
    [ros2_control_node-1] [INFO] [1739934409.437008655] [controller_manager]: Loading controller 'rrbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.438597455] [controller_manager]: Controller 'rrbot_joint_state_broadcaster' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_13/share/ros2_control_demo_example_13/config/three_robots_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.457075147] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Writing commands:
    [ros2_control_node-1] 	command pos: 0.00, vel: 0.00, acc: 0.00 for joint 0, control lvl: 2
    [ros2_control_node-1] 	command pos: 0.00, vel: 0.00, acc: 0.00 for joint 1, control lvl: 2
    [ros2_control_node-1] [INFO] [1739934409.457586882] [controller_manager]: Configuring controller: 'threedofbot_position_controller'
    [ros2_control_node-1] [INFO] [1739934409.460537377] [threedofbot_position_controller]: configure successful
    [ros2_control_node-1] [INFO] [1739934409.467136079] [controller_manager]: Configuring controller: 'rrbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.467235345] [rrbot_joint_state_broadcaster]: Publishing state interfaces defined in 'joints' and 'interfaces' parameters.
    [ros2_control_node-1] [WARN] [1739934409.467279858] [rrbot_joint_state_broadcaster]: Mapping from 'position' to interface 'position' will not be done, because 'position' is defined in 'interface' parameter.
    [ros2_control_node-1] [INFO] [1739934409.479895727] [controller_manager]: Activating controllers: [ rrbot_joint_state_broadcaster ]
Error: s2_control_node-1] [ERROR] [1739934409.487072719] [controller_manager]: Caught exception of type : St13runtime_error while claiming the state interfaces. Can't activate controller 'rrbot_joint_state_broadcaster': State interface with key 'rrbot_joint1/position' does not exist
Error: s2_control_node-1] [ERROR] [1739934409.488004367] [controller_manager]: Could not activate controller : 'rrbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934409.534343300] [controller_manager]: Loading controller : 'threedofbot_pid_gain_controller' of type 'forward_command_controller/ForwardCommandController'
    [ros2_control_node-1] [INFO] [1739934409.534394005] [controller_manager]: Loading controller 'threedofbot_pid_gain_controller'
    [ros2_control_node-1] [INFO] [1739934409.534740134] [controller_manager]: Controller 'threedofbot_pid_gain_controller' node arguments: --ros-args --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_3/share/ros2_control_demo_example_3/config/rrbot_multi_interface_forward_controllers.yaml --params-file /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/ros2_control_demo_example_13/share/ros2_control_demo_example_13/config/three_robots_controllers.yaml 
    [ros2_control_node-1] [INFO] [1739934409.559460011] [controller_manager]: Configuring controller: 'threedofbot_pid_gain_controller'
    [ros2_control_node-1] [INFO] [1739934409.560533572] [threedofbot_pid_gain_controller]: configure successful
    [ros2_control_node-1] [INFO] [1739934409.766734222] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Reading states:
    [ros2_control_node-1] 	pos: 0.00, vel: 0.00, acc: 0.00 for joint 0
    [ros2_control_node-1] 	pos: 0.00, vel: 0.00, acc: 0.00 for joint 1
    [spawner-4] [WARN] [1739934409.776926364] [spawner_joint_state_broadcaster]: [NON-XML-CHAR-0x1B][93mController already loaded, skipping load_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1739934409.778052203] [controller_manager]: Configuring controller: 'joint_state_broadcaster'
Error: s2_control_node-1] [ERROR] [1739934409.778120130] [controller_manager]: Controller 'joint_state_broadcaster' can not be configured from 'active' state.
Error: awner-4] [ERROR] [1739934409.779198206] [spawner_joint_state_broadcaster]: [NON-XML-CHAR-0x1B][91mFailed to configure controller[NON-XML-CHAR-0x1B][0m
Error: ROR] [spawner-4]: process has died [pid 46781, exit code 1, cmd '/home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/install/controller_manager/lib/controller_manager/spawner joint_state_broadcaster --ros-args'].
    [ros2_control_node-1] [INFO] [1739934409.957243386] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Writing commands:
    [ros2_control_node-1] 	command pos: 0.00, vel: 0.00, acc: 0.00 for joint 0, control lvl: 2
    [ros2_control_node-1] 	command pos: 0.00, vel: 0.00, acc: 0.00 for joint 1, control lvl: 2
    [INFO] [robot_state_publisher-2]: sending signal 'SIGINT' to process[robot_state_publisher-2]
    [INFO] [ros2_control_node-1]: sending signal 'SIGINT' to process[ros2_control_node-1]
    [robot_state_publisher-2] [INFO] [1739934410.165425873] [rclcpp]: signal_handler(signum=2)
    [ros2_control_node-1] [INFO] [1739934410.166060664] [rclcpp]: signal_handler(signum=2)
    [ros2_control_node-1] [INFO] [1739934410.166140612] [controller_manager]: Shutdown request received....
    [ros2_control_node-1] [INFO] [1739934410.166194192] [controller_manager]: Shutting down all controllers in the controller manager.
    [ros2_control_node-1] [INFO] [1739934410.166241852] [controller_manager]: Shutting down controller 'rrbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934410.166549980] [controller_manager]: Shutting down controller 'rrbot_with_sensor_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934410.166645078] [controller_manager]: Shutting down controller 'threedofbot_position_controller'
    [ros2_control_node-1] [INFO] [1739934410.166730688] [controller_manager]: Shutting down controller 'threedofbot_joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934410.166792294] [controller_manager]: Shutting down controller 'rrbot_with_sensor_position_controller'
    [ros2_control_node-1] [INFO] [1739934410.166843811] [controller_manager]: Deactivating controller 'joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934410.166973804] [controller_manager]: Shutting down controller 'joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1739934410.167030110] [controller_manager]: Shutting down controller 'threedofbot_pid_gain_controller'
    [ros2_control_node-1] [INFO] [1739934410.167084382] [controller_manager]: Deactivating controller 'forward_velocity_controller'
    [ros2_control_node-1] [INFO] [1739934410.167211190] [controller_manager]: Shutting down controller 'forward_velocity_controller'
    [ros2_control_node-1] [INFO] [1739934410.167290068] [resource_manager]: 'deactivate' hardware 'RRBotSystemMultiInterface' 
    [ros2_control_node-1] [INFO] [1739934410.167313181] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Deactivating... please wait...
    [INFO] [robot_state_publisher-2]: process has finished cleanly [pid 46708]
    [ros2_control_node-1] [INFO] [1739934411.167404822] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: 3.0 seconds left...
    [ros2_control_node-1] [INFO] [1739934412.167537668] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: 2.0 seconds left...
    [ros2_control_node-1] [INFO] [1739934413.167661989] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: 1.0 seconds left...
    [ros2_control_node-1] [INFO] [1739934413.167736068] [controller_manager.resource_manager.hardware_component.system.RRBotSystemMultiInterface]: Successfully deactivated!
    [ros2_control_node-1] [INFO] [1739934413.167753350] [resource_manager]: Successful 'deactivate' of hardware 'RRBotSystemMultiInterface'
    [ros2_control_node-1] [INFO] [1739934413.167764651] [resource_manager]: 'shutdown' hardware 'RRBotSystemMultiInterface' 
    [ros2_control_node-1] [INFO] [1739934413.167771114] [resource_manager]: Successful 'shutdown' of hardware 'RRBotSystemMultiInterface'
    [ros2_control_node-1] [INFO] [1739934413.167814024] [controller_manager]: Shutting down the controller manager.
Error: s2_control_node-1] [ERROR] [1739934413.167935231] [controller_manager.pal_statistics]: Exception in publisher thread: context cannot be slept with because it's invalid!. Aborting!
    [ros2_control_node-1] [INFO] [1739934413.176969606] [controller_manager.pal_statistics]: Async messages lost 0
    [ros2_control_node-1] [INFO] [1739934413.176984904] [controller_manager.pal_statistics]: publish_async_failures_ 0
    [INFO] [ros2_control_node-1]: process has finished cleanly [pid 46707]
    ----------------------------- Captured stderr call -----------------------------
    test_check_if_msgs_published (test_rrbot_system_multi_interface_launch.TestFixture.test_check_if_msgs_published) ... ok
    test_controller_running (test_rrbot_system_multi_interface_launch.TestFixture.test_controller_running) ... ok
    test_node_start (test_rrbot_system_multi_interface_launch.TestFixture.test_node_start) ... ok
    
    ----------------------------------------------------------------------
    Ran 3 tests in 1.818s
    
    OK
    test_exit_codes (test_rrbot_system_multi_interface_launch.TestDescriptionCraneShutdown.test_exit_codes)
    Check if the processes exited normally. ... FAIL
    
    ======================================================================
    FAIL: test_exit_codes (test_rrbot_system_multi_interface_launch.TestDescriptionCraneShutdown.test_exit_codes)
    Check if the processes exited normally.
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/src/ros-controls/ros2_control_demos/example_3/test/test_rrbot_system_multi_interface_launch.py", line 102, in test_exit_codes
        launch_testing.asserts.assertExitCodes(proc_info)
      File "/opt/ros/jazzy/lib/python3.12/site-packages/launch_testing/asserts/assert_exit_codes.py", line 62, in assertExitCodes
        assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    AssertionError: Proc spawner-4 exited with code 1
    
    ----------------------------------------------------------------------
```

--- a/example_13/test/test_three_robots_launch.py
+++ b/example_13/test/test_three_robots_launch.py
@@ -56,7 +56,7 @@ def generate_test_description():
                 "launch/three_robots.launch.py",
             )
         ),
-        launch_arguments={"gui": "false"}.items(),
+        launch_arguments={"gui": "false", "sigterm_timeout": "10"}.items(),
     )
 
     return LaunchDescription([launch_include, ReadyToTest()])

--- a/example_13/test/test_three_robots_launch.py
+++ b/example_13/test/test_three_robots_launch.py
@@ -30,6 +30,7 @@
 
 import os
 import unittest
+import psutil
 import subprocess
 
 from ament_index_python.packages import get_package_share_directory
@@ -349,3 +350,10 @@ class TestDescriptionCraneShutdown(unittest.TestCase):
     def test_exit_codes(self, proc_info):
         """Check if the processes exited normally."""
         launch_testing.asserts.assertExitCodes(proc_info)
+
+    def test_process_running(self):
+        for proc in psutil.process_iter():
+            # check whether the process name matches
+            if "ros2_control_node" in proc.name():
+                print("ros2_control_node found")
+                proc.kill()

